content-src/components/DebugPage/DebugPage.js

```diff
@@ -0,0 +1,35 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {dedupedSites} = require("selectors/selectors");
+
+const DebugPage = React.createClass({
+  render() {
+    return (<main className="debug-page">
+      <div className="new-tab-wrapper">
+        <h2>RAW STATE:</h2>
+        <pre>
+          {JSON.stringify(this.props.raw, null, 2)}
+        </pre>
+
+        <h2>DEDUPED NEW TAB STATE:</h2>
+        <pre>
+          {JSON.stringify(this.props.newTab, null, 2)}
+        </pre>
+      </div>
+    </main>);
+  }
+});
+
+module.exports = connect(state => {
+  return {
+    newTab: dedupedSites(state),
```

Comment:
Odd, when I run this locally, I'm seeing the following error in my DevTools console:

> **TypeError: dedupedSites is not a function**